Report/Chapters/Introduction.tex

```diff
@@ -2,7 +2,6 @@ \chapter{Introduction}
 
 Programming has always been an activity which is done on a computer with a keyboard. Despite the existence of different visual programming languages, like Blender, there are no programs in which well known programming languages like Java, Haskell or C(++) can be visually implemented. 
```

Comment:
Respectfully disagree: Blender is a 3D modeling tool. *One* way of using a *part* of that tool (the nodes interface to the compositor) has some superficial (visual) similarities to what we're doing, but I wouldn't call it a programming language, and doing so could perhaps confuse our readers.

A better example might be Pure Data (Pd), which is both niche enough for people never to have heard of, close enough to what we're doing, and far enough from something like Java for the sentence to make sense.